services/121-service/src/payments/fsp-integration/intersolve-voucher/intersolve-voucher.controller.ts

```diff
@@ -43,7 +43,12 @@ export class IntersolveVoucherController {
     private azureLogService: AzureLogService,
   ) {}
 
-  @AuthenticatedUser({ permissions: [PermissionEnum.PaymentVoucherREAD] })
+  @AuthenticatedUser({
+    permissions: [
+      PermissionEnum.PaymentVoucherPaperREAD,
+      PermissionEnum.PaymentVoucherWhatsappREAD,
```

Comment:
This makes BOTH required, right?
So users without the "`PaymentVoucherWhatsappREAD`" would not be able to export.

So, only "PaymentVoucherPaperREAD" would not be enough.